src/main/scala/slack/api/SlackApiClient.scala

```diff
@@ -921,7 +921,7 @@ class SlackApiClient private (token: String, slackApiBaseUri: Uri) {
   }
 
   def openIm(userId: String)(implicit system: ActorSystem): Future[String] = {
-    val res = makeApiMethodRequest("im.open", "user" -> userId)
+    val res = makeApiMethodRequest("conversations.open", "users" -> userId)
```

Comment:
ideally, it should accept a user list now, however it accepts one user just well (technically it's still a comma-separated list of one element 😄 )